crates/egui-winit/src/clipboard.rs

```diff
@@ -29,8 +29,8 @@ impl Clipboard {
     ///
     /// # Safety
     ///
-    /// The returned `Clipboard` must not outlive the input `_event_loop`.
-    pub fn new<T>(_event_loop: &EventLoopWindowTarget<T>) -> Self {
+    /// The returned `Clipboard` must not outlive the input `_display_target`.
+    pub fn new(_display_target: impl HasRawDisplayHandle) -> Self {
```

Comment:
could use `&dyn HasRawDisplayHandle` here and elsewhere to avoid monomorphization